client/src/components/PayBill.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useRef, useEffect, useState } from 'react';
+import $ from 'jquery';
+import ThankYou from './ThankYou.jsx';
+
+const payeeEmail = 'sb-dmus17722352@personal.example.com';
+const hostName = 'Milo';
+const billInfo = [
+  {
+    name: 'Milo',
+    order: [
+      { reference_id: 1, description: 'Milo"s order', amount: { currency_code: 'USD', value: 135.3 } }]
+  },
+  {
+    name: 'Sara',
+    order: [
+      { reference_id: 4, description: 'Sara"s order', amount: { currency_code: 'USD', value: 25.3 } }]
+  },
+  {
+    name: 'Tom',
+    order: [
+      { reference_id: 6, description: 'Tom"s order', amount: { currency_code: 'USD', value: 52.3 } }]
+  },
+  {
+    name: 'Jorge',
+    order: [
+      { reference_id: 5, description: 'Jorge"s order', amount: { currency_code: 'USD', value: 15.3 } }]
+  }
+];
+
+const BillCard = (props) => (
+  <div className='card bill-card' >
+    <input type="checkbox" className="btn-check" id={'btncheck' + props.index} autoComplete="off" onClick={props.onClick} />
+    <label className="btn btn-outline-danger" htmlFor={'btncheck' + props.index}>{props.info.name + ' $' + props.info.bill}</label>
+  </div>
+);
+
+const PayBill = (props) => {
+  const [submit, setSubmit] = useState(false);
+  const billInfoPay = billInfo.filter(info => info.name !== hostName);
+
+  const paypal = useRef();
+
+  const orderEnabledList = new Array(billInfoPay.length).fill(false);
+  console.log(orderEnabledList);
+  const onChange = (e) => {
+    const index = Number(e.target.id.split('btncheck')[1]);
+    orderEnabledList[index] = $('#' + e.target.id).is(':checked');
+  };
+  let orderToPay = [];
+  useEffect(() => {
+    window.paypal.Buttons(
+      {
+        createOrder: (data, actions, error) => {
+          for (let i = 0; i < orderEnabledList.length; i++) {
+            if (orderEnabledList[i]) {
+              orderToPay = orderToPay.concat(billInfoPay[i].order);
+            }
+          }
+          console.log(orderToPay);
+          return actions.order.create({
+            intent: 'CAPTURE',
+            purchase_units: orderToPay
+          });
+        },
+        onApprove: async (data, actions) => {
+          const order = await actions.order.capture();
+          $.ajax({
+            type: 'POST',
+            url: '/paypal',
+            data: { orderToPay: orderToPay, payeeEmail: payeeEmail },
+            success: (data) => {
+              console.log('success:' + data);
+            }
+          });
+          // const orderID = handleRequest(orderToPay, payeeEmail);
+          console.log(order);
```

Comment:
There's a few console.logs that aren't for errors, do we need them or can we clean them up?